v3.2/glfw/glfw/src/wl_window.c

```diff
@@ -24,8 +24,6 @@
 //
 //========================================================================
 
-#define _GNU_SOURCE
-
```

Comment:
@r0l1 I'm late to this party - is there are reason to diverge from upstream and remove `_GNU_SOURCE` here?

Context is: https://github.com/go-gl/glfw/pull/271.

Reply:
FYI I'm reinstating _GNU_SOURCE here, which documents what you should do if you want to remove it again: https://github.com/go-gl/glfw/pull/271/commits/bac7f0e2e1cc1b17dfc90d2c83c04f174e53acf8